### ISSUE TITLE
[Feature Request] Screenshot Formatting

### ISSUE THREAD

**Issue by korni2207** [Penwy/adv-ff#1]
### Feature Description

An ability to format filename of screenshots taken by OBS and its built-in feature to take screenshots. In Vanilla whenever you press set shortcut key to take a screenshot of the output source it creates a file formatted like this:

**Screenshot %CCYY_%MM-%DD %hh-%mm-%ss**

The script is gonna support Replay Buffer output file formatting in the future and would be great if the script could also support Screenshot output file formatting as well.

**Comment by Penwy** [comment]
I'll look into it.
Can't guarantee much, because as far as I remember, there's zero api interactions for screenshots, but I'll give it a look.

**Comment by chuman72486** [comment]
I agree that this would be helpful.  I take a lot of screenshots in OBS and being able to use scene and source names in the screenshots would be super helpful.
